petcare-sample/b2c/web-app/petdesk/web/react/public/config.js

```diff
@@ -5,7 +5,8 @@ window.config = {
   signOutRedirectURL: "http://localhost:3000",
   myAccountAppURL: "",
   resourceServerURL: "http://localhost:9090",
-billingServerURL: "http://localhost:9091",
-  scope: ["openid", "email", "profile"],
+  billingServerURL: "http://localhost:9091",
+  salesforceServerURL: "http://localhost:9092",
+  scope: ["openid", "email", "profile", "acr"],
```

Comment:
can we make the "resourceServerURL" as petManagementServiceURL

Reply:
Fixed with https://github.com/wso2/samples-is/pull/459/commits/9e5c58397065d90016b15074068651252e262b08